docs/docs/external/getting-started/test-network/setup-local.md

```diff
@@ -0,0 +1,337 @@
+    sudo apt-get install unzip
+    unzip protoc-3.15.6-linux-x86_64.zip -d <some-folder-path>
+    export PATH="$PATH:<some-folder-path>/bin"
+    go get -u google.golang.org/protobuf/cmd/protoc-gen-go
+    go get -u google.golang.org/grpc/cmd/protoc-gen-go-grpc
+    ```
+    _Note_: The latest version at present is `3.15.6`, but you should check the above link to find the most current version before running the above steps.
+
+### Credentials
+Make sure you have an SSH or GPG key registered in https://github.com to allow seamless cloning of repositories (at present, various setup scripts clone repositories using the `https://` prefix but this may change to `git@` in the future).
+
+## Getting the Code and Documentation
+
+Clone the [weaver-dlt-interoperability](https://github.com/hyperledger-labs/weaver-dlt-interoperability) repository. The code to get a basic test network up and running and test data-sharing interoperation flows lies in the subfolder `tests/network-setups`, which should be your starting point, though the setups will rely on other parts of the repository, as you will find out in the instructions given on this page.
+
+## Common Structures
+
+The `common/protos` folder contains structure definitions in the protobuf format that are used by all the different components. The various `common/protos-*` folders are meant to contain compiled protobufs (in different languages).
+
+To compile the protobufs for JavaScript, do the following:
+- Navigate to the `common/protos-js` folder.
+- Run the following command:
+  ```bash
+  make build
+  ```
+
+To compile the protobufs for Golang, do the following:
+- Navigate to the `common/protos-go` folder.
+- Run the following command:
+  ```bash
+  make build
+  ```
+
+To compile the protobufs for Java, do the following:
+- Navigate to the `common/protos-java-kt` folder.
+- Run the following command:
+  ```bash
+  make build
+  ```
+
+## Hyperledger Fabric Components
+
+Using the sequence of instructions below, you can start two separate Fabric networks, each with a single channel and application contract (chaincode). You can also start an interoperation contract, a relay and a _driver_ acting on behalf of each network. You can build a Fabric CLI tool with which you can initialize both networks' ledgers with access control policies, foreign networks' security groups (i.e., membership providers' certificate chains), and some sample key-value pairs that can be shared during subsequent interoperation flows.
+
+### Fabric Interoperation Node SDK
+
+A library, as companion to the `hyperledger/fabric-sdk-node`, is defined in the `sdks/fabric/interoperation-node-sdk` folder. This contains functions for Fabric Gateway-based applications to exercise interoperation capabilities via relays and also a number of utility/helper functions. The Fabric-CLI tool, which we will use later, depends on this library.
```

Comment:
also a number of utility/helper functions -> also several utility/helper functions

Reply:
Fixed